src/components/checkbox/checkbox.component.spec.ts

```diff
@@ -142,4 +142,57 @@ describe('Checkbox Component', () => {
         await fixture.whenStable();
     }
 
-});
\ No newline at end of file
+});
+
+@Component({
+    selector: 'checkbox-required-test',
+    template: `
+        <ux-checkbox
+            [required]="required"
+            [(value)]="checkModel.option1">
+        </ux-checkbox>
+    `
+})
+export class CheckboxRequiredTestComponent {
```

Comment:
I think you can safely merge the `required` property into one of the existing test components, like you did with the number picker.